extensions/amp-story/1.0/amp-story.js

```diff
@@ -405,11 +406,25 @@ export class AmpStory extends AMP.BaseElement {
 
     if (isExperimentOn(this.win, 'amp-story-branching')) {
       this.registerAction('goToPage', invocation => {
-        const {args} = invocation;
+        const {args, caller} = invocation;
+
         if (args) {
           this.storeService_.dispatch(
               Action.SET_ADVANCEMENT_MODE, AdvancementMode.GO_TO_PAGE);
-          this.switchTo_(args['id'], NavigationDirection.NEXT);
+
+          if (caller['offsetParent'].tagName === 'AMP-SIDEBAR') {
```

Comment:
1. Is it always exactly the parent?  Or are we really expecting it to be any ancestor?
2. Can we use `caller.offsetParent` instead of `caller['offsetParent']`?